src/panels/lovelace/editor/config-elements/glance-card/glance-options.ts

```diff
@@ -0,0 +1,38 @@
+import { ConfigEntity } from "../../../cards/hui-glance-card";
+
+export class GlanceOptions {
+  private entityConf: ConfigEntity;
+
+  constructor(entityConf: ConfigEntity) {
+    const defaults = {
+      name: "",
+      icon: "",
+      tap_action: "more-info" as "more-info",
```

Comment:
what to `as`?

Reply:
With out the `as` typescript doesnt like it. because we set tap_action in the interface to `"more-info" | "toggle" | "call-service"`

Reply:
https://github.com/Microsoft/TypeScript/issues/11465

Reply:
lame, but okay.